JS/Mery/ui-neumorphism-mery/example/README.md

```diff
@@ -0,0 +1,5 @@
+This example was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
This file is absolute copy of the source which we gave you. YOU SHOULD NOT COPY it fully
Delete this or change it by your own